src/wp-content/themes/twentytwentyone/assets/js/primary-navigation.js

```diff
@@ -115,7 +116,7 @@ function twentytwentyoneExpandSubMenu( el ) { // jshint ignore:line
 			mobileButton.onclick = function() {
 				wrapper.classList.toggle( id + '-navigation-open' );
 				wrapper.classList.toggle( 'lock-scrolling' );
-				twentytwentyoneToggleAriaExpanded( mobileButton );
+				twentytwentyoneToggleAriaExpanded( mobileButton, false );
```

Comment:
The function was getting static analysis complaints in PhpStorm since it was not provided. The function is using the falsiness of the `undefined` value.